CacheAdvance.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name     = 'CacheAdvance'
-  s.version  = '1.1.0'
+  s.version  = '1.2.0'
```

Comment:
We're technically changing public API in the error enum case. But maybe that's small enough that it's OK?

Reply:
It's not a breaking change unless you're creating your own error... which... I don't think that's worth supporting with semver? A `catch CacheAdvanceError.incompatibleHeader` clause still works. Willing to roll a major version though since you're right that this is indeed a breaking change in some cases.

Reply:
I am OK keeping this a minor version since we wouldn't expect someone to be instantiating one of these errors.